v22.2/ui-insights-page.md

```diff
@@ -0,0 +1,210 @@
+
+The following screenshot shows the statement execution of the query described in [Use the right index]({{ link_prefix }}apply-statement-performance-rules.html#rule-2-use-the-right-index):
+
+<img src="{{ 'images/v22.2/statement_executions.png' | relative_url }}" alt="Statement execution" style="border:1px solid #eee;max-width:100%" />
+
+To view details of the execution, click an execution ID in the **Statement Execution ID** column.
+
+#### Statement Execution Details
+
+The **Statement Execution Details** view provides more details on a statement execution insight. Additional information includes:
+
+- **Start Time**: The time the statement execution started.
+- **End Time**: The time the statement execution ended.
+- **Elapsed Time**: The time that elapsed during statement execution.
+- **Rows Read**: The total number of rows read.
+- **Rows Written**: The total number of rows written.
+- **Transaction Priority**: The [priority](transactions.html#set-transaction-priority) of the transaction for the statement execution.
+- **Full Scan**: Whether the execution performed a full scan of the table.
+- **Session ID**: The ID of the [session](ui-sessions-page.html) the statement was executed from.
+- **Transaction Fingerprint ID**: The ID of the transaction fingerprint for the statement execution.
+- **Transaction Execution ID**: The ID of the transaction execution for the statement execution.
+- **Statement Fingerprint ID**: The fingerprint ID of the statement fingerprint for the statement execution..
+
+The following screenshot shows the execution details of the statement execution in the preceding section:
+
+<img src="{{ 'images/v22.2/statement_execution_details.png' | relative_url }}" alt="Statement execution details" style="border:1px solid #eee;max-width:100%" />
+
+The Insights column shows the name of the insight, in this case **Suboptimal Plan**, the Details column provides details on the insight, and the final column contains a button to perform a query to mitigate the cause of the insight,
+in this case to create an index on the ride start time that stores the rider ID.
+
+## Schema Insights view
+
+To display this view, click **Insights** in the left-hand navigation of the DB Console and select **Schema Insights**.
+
+This view lists the indexes that have not been used and should be dropped and or the ones that should be created, altered, or replaced (based on statement execution).
+
+- The drop recommendations are the same as those on the [Databases](ui-databases-page.html) page.
+- The create, alter, and replace recommendations are the same as those on the [Explain Plans tab](ui-statements-page.html#insights) on the Statements page. The Explain Plans tab shows all recommendations, however this page shows only the latest recommendations for that statement fingerprint. If you execute a statement again after creating or updating an index, the recommendation disappears.
```

Comment:
"If you execute a statement again after creating or updating an index, the recommendation disappears."

I dont think this is the case today right @maryliag I remember seeing the recommendation still showing up on the insights page. If so, we should call out this behavior. We don't have validation yet to determine if a recommendation has been applied.